test/corectl_integration_test.go

```diff
@@ -41,6 +41,8 @@ func TestReload(t *testing.T) {
 	p := toolkit.Params{T: t, Config: "test/projects/using-entities/corectl.yml", Engine: *toolkit.EngineStdIP, App: t.Name()}
 	defer p.Reset()
 	p.ExpectOK().Run("build")
+	p.ExpectIncludes("<<  3 Lines fetched").Run("build", "--limit", "3")
```

Comment:
Can we also add a test case verifying the nbr of lines fetched when not passing the `limit` flag?

Reply:
Fixed